### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by lievenhey**

### CODE REVIEW

**Comment by GitMensch** [comment]
Possibly missing: hint at the 'new' "go to" command via context menu if there is not a totally clean way to reach it that I've missed.

Do I misremember that the arrows were drawn as lines instead of ascii?

File: README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ The main feature of hotspot is visualizing a `perf.data` file graphically.
 
 ![hotspot FlameGraph page](screenshots/flamegraph.png?raw=true "hotspot FlameGraph page")
 
+An inlined functions has a darker border compared to a normal function.
```
[the remark this answers is not in the document]
Comment (milianw) [reply README.md:59]: ack, functions -> function, or better yet rephrase to:

Note: Inlined functions have a darker border than non-inlined ones.

**Comment by GitMensch** [comment]
Possibly add to the command line options as well after merging #653.

**Comment by lievenhey** [comment]
> Do I misremember that the arrows were drawn as lines instead of ascii?

That is a feature that @milianw added some time ago

**Review by GitMensch — CHANGES_REQUESTED** [review]
English messages - then fine to merge.

Thank you for updating the screenshots to have the nice lines in as well.

Concerning the flamegraph - maybe some of the perf.data testfiles in the repo look "more interesting" while still showing the darker (inlined) frames?

**Comment by lievenhey** [comment]
:facepalm:

**Review by milianw — APPROVED** [review]

**Pull request merged**